runtime/near-wallet-contract/src/lib.rs

```diff
@@ -12,6 +14,15 @@ static MAINNET: WalletContract =
 static TESTNET: WalletContract =
     WalletContract::new(include_bytes!("../res/wallet_contract_testnet.wasm"));
 
+/// Initial version of WalletContract. It was released to testnet, but not mainnet.
+/// We still use this one on testnet protocol version 70 for consistency.
+static OLD_TESTNET: WalletContract =
+    WalletContract::new(include_bytes!("../res/wallet_contract_testnet_old.wasm"));
```

Comment:
```suggestion
    WalletContract::new(include_bytes!("../res/wallet_contract_testnet_70.wasm"));
```

or some other mechanism by which it is obvious what protocol versions these contracts refer to. (Change the procotol version number to the one on which the contract has become available.)

Reply:
Done in https://github.com/near/nearcore/pull/11975/commits/aeb11a9fe8bd9c66ff4c1e1af046f3d8c5d57b58